NZBHydra2.json

```diff
@@ -0,0 +1,47 @@
+{
+    "NZBHydra2": {
+        "containers": {
+            "nzbhydra2": {
+                "image": "linuxserver/nzbhydra2",
+                "launch_order": 1,
+                "ports": {
+                    "5076": {
+                        "description": "NZBHydra2 WebUI port. Suggested default: 5076",
+                        "host_default": 5076,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for NZBHydra2 configuration. Eg: create a Share called nzbhydra-config for this purpose alone.",
+                        "label": "Config Storage"
+                    },
+                    "/downloads": {
+                        "description": "Choose a Share for NZBHydra downloads. Eg: create a Share called nzbhydra-downloads for this purpose alone.",
+                        "label": "Download Storage"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid UID to run nzbhydra as. It must have full permissions to all Shares mapped in the previous step.",
+                        "label": "UID to run NZBHydra as.",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid GID to use along with the above UID. It (or the above UID) must have full permissions to all Shares mapped in the previous step.",
+                        "label": "GID to run NZBHydra as.",
+                        "index": 2
+                    }
+                }
+            }
+        },
+        "description": "NZBHydra2 is a meta search for NZB indexers. <p>Uses custom docker image by <a href='https://www.linuxserver.io/' target='_blank'>Linuxserver.io</a>: <a href='https://hub.docker.com/r/linuxserver/nzbhydra2/' target='_blank'>https://hub.docker.com/r/linuxserver/nzbhydra2/</a>, available for amd64, arm32v7 and arm64v8.</p>",
+        "ui": {
+            "slug": ""
+        },
+        "volume_add_support": false,
```

Comment:
@thedrjones , is there a good reason to prevent the addition of volumes here? I'm not a user of this project so I don't know how it works, but do you think some users might want to bind their existing media library, for instance? If yes, we might want to allow that. Let us know what you think.

Reply:
I have never needed to bind another set of shares to it over the years, so that's why I never enabled it :).
NZBHydra is just a search aggregator that allows you to combine searches on your configured newsgroup indexers (e.g sites like NewsBin, NZBGeek) through one unified web page (rather than checking out several indexers by hand for the same search, it can then weight those results to things you may consider more reliable higher up the list).
It can also allow you to send the result nzb metadata file over to your newsgroup client (e.g. SABnzbd) for that client to do the download. NZBHydra itself does not actually need to download into or touch your media libraries.

Reply:
Makes sense, thanks a lot for the information!